src/baxter_interface/camera.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, name):
 
         list_svc = rospy.ServiceProxy('/cameras/list', ListCameras)
         rospy.wait_for_service('/cameras/list', timeout=10)
-        if not self._id in list_svc().cameras:
+        if not self._id in ["left_hand_camera","right_hand_camera","head_camera"]:
```

Comment:
In the old SDK a roservice call /cameras/list listed you all available cameras. In the new version you get exactly the opened cameras which are left_hand_camera and right_hand_camera by default.